config/config.py

```diff
@@ -0,0 +1,3 @@
+from config.sah_config import SAHConfig, get_db_credentials_path
+
+sah_config = SAHConfig(credentials_path=get_db_credentials_path())
```

Comment:
Happy to have it elsewhere too, but it just seems like a lot to have a file just for these two lines.

Reply:
ok, tried to move it back into app.py and it created a metric tonne of circular imports, mainly because app.py imports create_app.py so it can create the app but if the config is in app.py then create_app.py needs to import it from there, hence the circle.

We can definitely move it somewhere else, but I can't think of anywhere that makes more sense unless it's a similar file just in the root or a different folder, which kinda defeats the point of moving it. Open to suggestions, though?